docs/01.GETTING-STARTED.md

```diff
@@ -21,51 +21,128 @@ To make our scripts run correctly, several shell utilities should be available o
 
 - `find`
 - `awk`
-- `wget`
 
-### Building Debug Version
+### Building Jerryscript
 
-To build debug version for Linux:
+##### To build debug version for Linux:
 
 ```bash
 python tools/build.py --debug
 ```
 
-To build debug version for Linux without LTO (Link Time Optimization):
+##### To build debug version for Linux without LTO (Link Time Optimization):
 
 ```bash
 python tools/build.py --debug --lto=off
 ```
 
-Add custom arguments to CMake:
+##### Add custom arguments to CMake:
 
 ```bash
-python tools/build.py --cmake-param CMAKE_PARAM
+python tools/build.py --cmake-param=CMAKE_PARAM
 ```
 
-Add toolchain file:
+##### Set an ECMAScript profile mode (full|compact|minimal):
 
-The cmake dir already contains some usable toolchain files.
+```bash
+python tools/build.py --feature=full|compact|minimal
+```
+
+##### Use (jerry|compiler-default|external libc) libc:
+
+The default libc is jerry-libc, but you can use compiler-default libc or an external libc:
+- compiler-default libc:
+
+```bash
+python tools/build.py --jerry-libc=off --compiler-default-libc=on
```

Comment:
What happens if --jerry-libc=off and --compiler-default-libc=off? Wouldn't be better if disabling jerry libc would enable compiler libc automatically?